python/stempy/image/__init__.py

```diff
@@ -8,8 +8,15 @@ def create_stem_images(reader, inner_radii,
                                      inner_radii, outer_radii, width, height,
                                      center_x, center_y)
 
-    images = [np.array(img, copy=False) for img in imgs]
-    return np.array(images, copy=False)
+    # images = [np.array(img, copy=False) for img in imgs]
+    # return np.array(images, copy=False)
+    return imgs
```

Comment:
This change breaks the `stem_image.py` example, and possibly some other scripts as well. It'd probably be preferable to not change this function. Or, if it is needed, modify all the other scripts so that they work with the changes.

Reply:
Looks like it breaks the `create_hdf5.py` example also.

Reply:
I am sorry that I forgot to modify other example scripts. 

I think the change is beneficial - not only to my function which takes STEMImage as input type, but also would make future post-processings easier. I personally think that, since everything is computed through type `STEMImage` in C++, Numpy array is only useful when saving or displaying result in Python, and should be an "extra" step in Python rather than wrapping inside a function.

Reply:
I have made the corresponding changes in example scripts that include function `create_stem_images()`. 
Additional lines of either (depends on if single or multiple STEM images have been returned)
`img=np.array(img, copy=False)` 
or 
`imgs = [np.array(img, copy=False) for img in imgs]`
`imgs = np.array(imgs, copy=False)`
has been added.

Reply:
Okay, sounds good. Thank you!

Reply:
Why not just create another function? You are pushing more boiler plate code to the outside, when you could create a new wrapper with the old name that simply does this conversion to a NumPy array. Making an additional function that returns the imgs you want, and then having a simple forwarding function that returns a NumPy array would give us the best of both worlds. Users from the Python side will almost always want the NumPy array, and should get it from our API.

Reply:
@cryos That indeed sounds like a better idea! I'm working on it right now, thanks.